### ISSUE TITLE
"Repository not found" after profile switch

### ISSUE THREAD

**Issue by inqrt** [gabe565/gh-profile#28]
When I switch from my profile A to profile B and for example do a `git push`, I get the error message:

```
remote: Repository not found.
fatal: repository 'https://github.com/<myuser>/<repo>.git/' not found
```

Is this related to the change in token storage by `gh` (because there is no `token` setting in host.yml anymore) and how can I fix this?

**Comment by gabe565** [comment]
@inqrt Yes, this is related to secure token storage. The gh cli does not yet have a way to change this token. I'm working with the gh team to add support, but for now you will need to login with `gh auth login --insecure-storage`

**Comment by inqrt** [comment]
Thanks man, the quick response is much appreciated! Also big kudos for the great job you do on the profile switcher extension 👌

**Comment by inqrt** [comment]
Sorry,  need to reopen this: I've noticed that even with `--insecure-storage`, after switching to another profile the repo url cannot be found when for example doing a `git pull` (both my profiles have been reauthenticated with the parameter)

**Comment by gabe565** [comment]
Thanks for the kudos! 🙂 

Hmm that's not good to hear. Can you send the output of `gh --version` and `gh profile --version`?

**Comment by gabe565** [comment]
Oh also, which operating system are you running?

**Comment by elschubi** [comment]
Sorry, I've missed your reply - it seems that it is working now (problem was on MacOS btw), at least after updating gh and reauthenticating all account with `--insecure-storage` I'm positive :)

**Comment by inqrt** [comment]
Sorry, missed your reply - seems to be working now (OS was MacOS btw). I've updated gh and re-authenticated with `--insecure-storage` - thanks!
